src/teaching-element/Assessment/SingleChoice.vue

```diff
@@ -3,30 +3,32 @@
     <span class="form-label">Solution</span>
     <ul class="answers">
       <li
-        v-for="(answer, index) in answers"
-        :key="index"
-        :class="{ selected: isSelected(index) }">
+        v-for="({answer, id}, index) in mappedAnswers"
+        :key="id"
+        :class="getAnswerClass(index)">
         <input
           v-model="userAnswer"
+          :id="id"
           :value="index"
           :disabled="disabled"
           @change="update"
           class="answers-radio"
           type="radio">
-        <span class="order">{{ transform(index) }}.</span>
-        <span>{{ answer }}</span>
+        <label :for="id">{{ transform(index) }}. {{ answer }}</label>
       </li>
     </ul>
   </div>
 </template>
 
 <script>
+import cuid from 'cuid';
 import { rules } from '../../util/listingType';
```

Comment:
Order is not correct.

Reply:
You sure?

Reply:
My bad. I saw the `defaults` below and thought they were an `import` as well. Sorry about that. Just put an empty line before `defaults`.